Closes #151 Council update (#153)
pharmaverse/blog: posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd

---
title: "Appsilon and Sanofi joins the pharmaverse council!"
author:
  - name: Ari Siggaard Knoph
description: "Updates to the pharmaverse council"
date: "2024-04-29"
# please do not use any non-default categories.
# You can find the default categories in the repository README.md
categories: [community]
# feel free to change the image
image: "pharmaverse.png"

---

<!--------------- typical setup ----------------->

```{r setup, include=FALSE}
long_slug <- "2024-04-29_appsilon_and__sa..."
# renv::use(lockfile = "renv.lock")
```

<!--------------- post begins here ----------------->

> We are excited to announce that **Appsilon** and **Sanofi** will take up the remaining two council seats on the pharmaverse council!

Reviewer: ```suggestion
We are excited to announce that **Appsilon** and **Sanofi** will take up the remaining two open council seats on the pharmaverse council!
```

Author: Accepted this one in push

Appsilon will on the council be represented by **[Damian Rodziewicz](https://www.linkedin.com/in/drodziewicz/)** and Sanofi by **[Andre Couturier](https://www.linkedin.com/in/andre-couturier-1800b447/)**.

> # Appsilon and Open Source

Reviewer: could be worth adding each company's logo just for some imagery to mix up the blog

> Appsilon is the maintainer of the [`rhino`](https://appsilon.github.io/rhino/) package already on the pharmaverse package rooster. Appsilon also powers the popular [ShinyConf](https://www.shinyconf.com/) and are involved in the pilot submissions to FDA through the R consortium working group ["R for submission"](https://rconsortium.github.io/submissions-wg/).

Reviewer: ```suggestion
Appsilon is the maintainer of the [`rhino`](https://appsilon.github.io/rhino/) package already on the pharmaverse package roster, as well as having numerous employees contributing to pharmaverse in spaces such as TLG packages and our [Examples site](https://pharmaverse.github.io/examples/). Appsilon also powers the popular [ShinyConf](https://www.shinyconf.com/) and are involved in the pilot submissions to FDA through the R consortium working group ["R for submission"](https://rconsortium.github.io/submissions-wg/).
```

Author: Accepted this one in push

# Sanofi and Open Source

> Sanofi recently became a member of the R consortium and have in recent years invested heavily in the adoption of R internally. In line with the pharmaverse goal, Sanofi is committed to the adoption and further development of open source software in the pharma industry. This is evident through Sanofi's contributions to [`falcon`](https://pharmaverse.github.io/falcon/) and [`riskmetric`](https://pharmar.github.io/riskmetric/) and their [extension](https://phuse.s3.eu-central-1.amazonaws.com/Archive/2024/Connect/US/Bethesda/PAP_ET10.pdf) of the [`teal`](https://insightsengineering.github.io/teal/latest-tag/) framework.

Reviewer: ```suggestion
Sanofi recently became a member of the R consortium and have in recent years invested heavily in the adoption of R internally. In line with the pharmaverse goal, Sanofi is committed to the adoption and further development of open source software in the pharma industry. This is evident through Sanofi's contributions to [`falcon`](https://pharmaverse.github.io/falcon/) and [`riskmetric`](https://pharmar.github.io/riskmetric/) and their [extension](https://phuse.s3.eu-central-1.amazonaws.com/Archive/2024/Connect/US/Bethesda/PAP_ET10.pdf) of the [`teal`](https://insightsengineering.github.io/teal/latest-tag/) framework, which they are working towards open sourcing.
```

Author: Accepted this one in push

> 

Reviewer: Maybe add a small section on why we accepted them, referencing the section on our site about expectations of council members, just to remind people of the high bar it takes to be a council member, and where to look for more details on what being on the council entails.

Author: Re-iterated what the companies are committing to with a link to our charter.
